cardano-node/src/Cardano/Tracing/Config.hs

```diff
@@ -21,78 +28,185 @@ data TraceOptions
   | TracingOn TraceSelection
   deriving (Eq, Show)
 
+type TraceAcceptPolicy = ("TraceAcceptPolicy" :: Symbol)
+type TraceBlockchainTime = ("TraceBlockchainTime" :: Symbol)
+type TraceBlockFetchClient = ("TraceBlockFetchClient" :: Symbol)
+type TraceBlockFetchDecisions = ("TraceBlockFetchDecisions" :: Symbol)
+type TraceBlockFetchProtocol = ("TraceBlockFetchProtocol" :: Symbol)
+type TraceBlockFetchProtocolSerialised = ("TraceBlockFetchProtocolSerialised" :: Symbol)
+type TraceBlockFetchServer = ("TraceBlockFetchServer" :: Symbol)
+type TraceChainDB = ("TraceChainDB" :: Symbol)
+type TraceChainSyncClient = ("TraceChainSyncClient" :: Symbol)
+type TraceChainSyncBlockServer = ("TraceChainSyncBlockServer" :: Symbol)
+type TraceChainSyncHeaderServer = ("TraceChainSyncHeaderServer" :: Symbol)
+type TraceChainSyncProtocol = ("TraceChainSyncProtocol" :: Symbol)
+type TraceDnsResolver = ("TraceDnsResolver" :: Symbol)
+type TraceDnsSubscription = ("TraceDnsSubscription" :: Symbol)
+type TraceErrorPolicy = ("TraceErrorPolicy" :: Symbol)
+type TraceForge = ("TraceForge" :: Symbol)
+type TraceForgeStateInfo = ("TraceForgeStateInfo" :: Symbol)
+type TraceHandshake = ("TraceHandshake" :: Symbol)
+type TraceIpSubscription = ("TraceIpSubscription" :: Symbol)
+type TraceKeepAliveClient = ("TraceKeepAliveClient" :: Symbol)
+type TraceLocalChainSyncProtocol = ("TraceLocalChainSyncProtocol" :: Symbol)
+type TraceLocalErrorPolicy = ("TraceLocalErrorPolicy" :: Symbol)
+type TraceLocalHandshake = ("TraceLocalHandshake" :: Symbol)
+type TraceLocalTxSubmissionProtocol = ("TraceLocalTxSubmissionProtocol" :: Symbol)
+type TraceLocalTxSubmissionServer = ("TraceLocalTxSubmissionServer" :: Symbol)
+type TraceLocalStateQueryProtocol = ("TraceLocalStateQueryProtocol" :: Symbol)
+type TraceMempool = ("TraceMempool" :: Symbol)
+type TraceMux = ("TraceMux" :: Symbol)
+type TraceTxInbound = ("TraceTxInbound" :: Symbol)
+type TraceTxOutbound = ("TraceTxOutbound" :: Symbol)
+type TraceTxSubmissionProtocol = ("TraceTxSubmissionProtocol" :: Symbol)
+
+newtype OnOff (name :: Symbol) = OnOff { isOn :: Bool } deriving (Eq, Show)
+
+class TracerName tm where
+    getName :: tm -> Text
```

Comment:
@CodiePP you might now better if having such a class is useful for other context or not, if just for the `OnOff` type I would prefer just a function.